IM_calculation/Advanced_IM/Models/Spear_3D_Model/run.py

```diff
@@ -0,0 +1,311 @@
+import argparse
+import glob
+import os
+import pandas as pd
+import subprocess
+
+import numpy as np
+
+
+DEFAULT_OPEN_SEES_PATH = "OpenSees"
+
+model_dir = os.path.dirname(__file__)
+
+
+def main():
+
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "comp_000", help="filepath to a station's 000 waveform ascii file"
+    )
+    parser.add_argument(
+        "comp_090", help="filepath to a station's 090 waveform ascii file"
+    )

```

Comment:
having them separate arguments provide flexibility. the script/wrapper above this handle the naming things.
this will allow us to run scripts when waveforms at extracted into tmp folders (which we currently already does) and it will not break if waveforms are in different tmp.

Reply:
I don't like assumptions. My thoughts are the current way is better, may be more verbose but this args are behind the scenes anyway.